pkg/services/meta/meta.go

```diff
@@ -32,6 +33,17 @@ type ContainerLister interface {
 	List() (map[cid.ID]struct{}, error)
 }
 
+// wsClient is for test purposes only.
+type wsClient interface {
+	GetBlockNotifications(blockHash util.Uint256, filters ...*neorpc.NotificationFilter) (*result.BlockNotifications, error)
```

Comment:
suggest to write docs incl. what behavior `Meta` expects for blocks w/o notifications - error of both nil?

Reply:
what docs? this is literally simplification for tests. if there is either a test http server that can be mocked or we do not use tests at all, this interface does not exist. this is literally `rpcclient.WSClient`. docs will not save us if neo-go decides to change behavior. i do not plan to have any self written implementation here ever

Reply:
ik why it's needed, idk how `GetBlockNotifications` interface behaves in mentioned case and wanna highlight this

Reply:
will some reference to `rpcclient.WSClient` be enough in the comment? cause i do not understand you. nothing should implement it and it declares nothing for no one, it is literally `rpcclient.WSClient` but it is hard to mock it for tests. and i prefer this interface not to exist, but it would decrease coverage. i believe we will come to `httptest` package usage but not now

Reply:
yeah, u may state that this is a part of `WSClient` interface consumed by `Meta`